src/web/app/components/sortable-table/sortable-table.component.html

```diff
@@ -2,7 +2,7 @@
   <thead>
     <tr>
       <th *ngFor="let column of columns" [ngClass]="{'sortable-header': column.sortBy}" (click)="column.sortBy && onClickHeader(column.header)">
-        <span [ngbTooltip]=column.headerToolTip container="body">{{ column.header }}</span>
+        <span [ngbTooltip]=column.headerToolTip container="body">{{ column.header }} </span>
```

Comment:
You don't want unnecessary change like this?

Reply:
I forgot to mention in the title, but this makes the layout looks better, with a spacing between the header and the arrows.

Old one was something like this

![a](https://user-images.githubusercontent.com/32454748/84394553-ac288b00-ac2f-11ea-905c-175774dcdeb5.gif)

Reply:
I see. Then I think a proper fix is to use CSS style rather than a space.

Reply:
Ok. Will change to that.